### PR TITLE
Remove uv_shutdown because FIN is already sent during uv_close

### PR DESCRIPTION
**Pull request by davidfowl**
I also ran a manual test with a linux VM and windows client with the following code:

Client

```C#
Socket socket = new Socket(SocketType.Stream, ProtocolType.Tcp);
socket.Connect(new IPEndPoint(IPAddress.Parse(<IP>), 5000));
var request = Encoding.ASCII.GetBytes("GET / HTTP/1.1\r\nHost:\r\n\r\n");
socket.Send(request);
var result = socket.Receive(new byte[32]);
Console.WriteLine(result);
```

Server

```C#
app.Run(context =>
{
    context.Abort();
    return Task.CompletedTask;
});
```

It prints out 0

Also confirmed it in Wireshark:

image

#1808

### CODE REVIEW

**Comment by cesarbs** [comment]
Add a unit test.

File: src/Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv/Internal/LibuvOutputConsumer.cs [hunk 1]
```diff
@@ -67,18 +67,6 @@ public class LibuvOutputConsumer
 
                     if (result.IsCancelled)
                     {
-                        // Send a FIN
-                        _log.ConnectionWriteFin(_connectionId);
-
-                        using (var shutdownReq = new UvShutdownReq(_log))
```
Comment (cesarbs) [inline src/Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv/Internal/LibuvOutputConsumer.cs]: I think this is the only place where we use `UvShutdownReq`. Can we remove the type altogether? Or do you prefer to keep it around?

Comment (davidfowl) [inline src/Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv/Internal/LibuvOutputConsumer.cs]: I prefer to keep it around,

**Comment by cesarbs** [comment]
:rage1:

**Comment by davidfowl** [comment]
:rage4:

**Comment by cesarbs** [comment]
🚢 when green.

**Pull request merged**